### PR TITLE
TINY-11405: improve dark skin for tiny comments

### PR DESCRIPTION
**Pull request by ztomaszyk**
Related Ticket: TINY-11405

Description of Changes:
Changed dark mode according to designs. 

Pre-checks:
* [x] ~~Changelog entry added~~
* [x] ~~Tests have been added (if applicable)~~
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] ~~Docs ticket created (if applicable)~~

GitHub issues (if applicable):

### CODE REVIEW

File: modules/oxide/src/less/skins/content/dark/content.less [hunk 1]
```diff
@@ -85,3 +90,62 @@ code {
...
+    & span.mce-preview-object {
+      &:not([data-mce-selected]) {
+        outline: @document-comment-outline-width solid @document-comment-active-outline-color;
+      }
+    }
+  }
+
+  span.tox-comment:not([data-mce-selected]) {
+    background-color: @document-comment-background-color;
+    outline: none;
```
Comment (spocke) [inline modules/oxide/src/less/skins/content/dark/content.less:139]: I would expect only color overrides in this skin file. Is this needed to be set here isn't the base skin setting this already?

File: modules/oxide/src/less/skins/content/dark/content.less [hunk 2]
```diff
@@ -85,3 +90,62 @@ code {
...
+  }
+
+  span.tox-comment[data-mce-annotation-active="true"]:not([data-mce-selected="inline-boundary"]) {
+    background-color: @document-comment-active-background-color;
+  }
+
+  span.tox-comment[data-mce-selected] {
+    background-color: @document-comment-active-background-color;
+    border-radius: 3px;
```
Comment (spocke) [inline modules/oxide/src/less/skins/content/dark/content.less:148]: Is this set in the base skin?

File: .changes/unreleased/tinymce-TINY-11405-2024-10-20.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: 'Improved dark skin for Tiny Comments plugin.'
```
Comment (spocke) [inline .changes/unreleased/tinymce-TINY-11405-2024-10-20.yaml:3]: Not sure where we should stick this comment it's a fix for the comments plugin but it needs to be in core. 🤔

**Review by spocke — APPROVED** [review]

File: .changes/unreleased/tinymce-TINY-11405-2024-10-20.yaml (hunk 3 above)
Comment (ztomaszyk) [reply .changes/unreleased/tinymce-TINY-11405-2024-10-20.yaml]: Removed ✅

**Review by lorenzo-pomili — APPROVED** [review]

**Pull request merged**